### PR TITLE
Warbling Earwigs

### PR DESCRIPTION
**Pull request by Geevies**

### CODE REVIEW

File: code/game/objects/items/skrell.dm [hunk 1]
```diff
@@ -153,3 +153,57 @@
 
 		if(hologram_message)
 			visible_message("<span class='notice'>[hologram_message]</span>")
+
+/obj/item/jargon_loyalty_implant
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
/obj/item/jargontag
```

File: code/game/objects/items/skrell.dm [hunk 2]
```diff
@@ -153,3 +153,57 @@
...
+	desc = "An ear-tag that shows the wearer is loyal to the Jargon Federation. A small cable travels into the ear canal..."
+	w_class = ITEMSIZE_SMALL
+	slot_flags = SLOT_EARS
+	icon = 'icons/obj/skrell_items.dmi'
+	icon_state = "jargtag"
+	item_state = "jargtag"
+	contained_sprite = TRUE
+	var/fried = FALSE // Doesn't work anymore
+
+/obj/item/jargon_loyalty_implant/equipped(mob/living/carbon/human/M)
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
/obj/item/jargontag/equipped(mob/living/carbon/human/M)
```

File: code/game/objects/items/skrell.dm [hunk 3]
```diff
@@ -153,3 +153,57 @@
...
+	..()
+	if(fried)
+		return
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.l_ear == src || H.r_ear == src)
+			clamp_on(H)
+
+// Could add some stuff to this in the future? I dunno. I just couldn't figure out how to callback to_chat LOL - geeves
+/obj/item/jargon_loyalty_implant/proc/do_loyalty(var/mob/wearer)
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
/obj/item/jargontag/proc/do_loyalty(var/mob/wearer)
```

File: code/game/objects/items/skrell.dm [hunk 4]
```diff
@@ -153,3 +153,57 @@
...
+	to_chat(wearer, span("good", "You feel an intense feeling of loyalty towards the Jargon Federation surge through your brain."))
+
+/obj/item/jargon_loyalty_implant/proc/clamp_on(var/mob/wearer)
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
/obj/item/jargontag/proc/clamp_on(var/mob/wearer)
```

File: code/game/objects/items/skrell.dm [hunk 5]
```diff
@@ -153,3 +153,57 @@
...
+	if(fried)
+		return
+	anchored = TRUE // good luck getting this bad boy off
+	canremove = FALSE
+	icon_state = "[initial(icon_state)]_active"
+	to_chat(wearer, span("warning", "\The [src] clamps down around your ear, releasing a burst of static before going silent. Something probes at your ear canal..."))
+	addtimer(CALLBACK(src, .proc/do_loyalty, wearer), 15)
+
+/obj/item/jargon_loyalty_implant/proc/unclamp()
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
/obj/item/jargontag/proc/unclamp()
```

File: code/game/objects/items/skrell.dm [hunk 6]
```diff
@@ -153,3 +153,57 @@
...
+	if(fried)
+		return
+	if(anchored && !canremove)
+		icon_state = initial(icon_state)
+		visible_message(span("warning", "\The [src] fizzles loudly, then clicks open!"))
+		anchored = FALSE
+		canremove = TRUE
+		fried = TRUE
+
+/obj/item/jargon_loyalty_implant/emp_act(severity)
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
/obj/item/jargontag/emp_act(severity)
```

File: code/game/objects/items/skrell.dm [hunk 7]
```diff
@@ -153,3 +153,57 @@
...
+	unclamp()
+
+/obj/item/jargon_loyalty_implant/emag_act(var/remaining_charges, var/mob/user)
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
/obj/item/jargontag/emag_act(var/remaining_charges, var/mob/user)
```

File: code/game/objects/items/skrell.dm [hunk 8]
```diff
@@ -153,3 +153,57 @@
...
+			clamp_on(H)
+
+// Could add some stuff to this in the future? I dunno. I just couldn't figure out how to callback to_chat LOL - geeves
+/obj/item/jargontag/proc/do_loyalty(var/mob/wearer)
+	to_chat(wearer, span("good", "You feel an intense feeling of loyalty towards the Jargon Federation surge through your brain."))
+
+/obj/item/jargontag/proc/clamp_on(var/mob/wearer)
+	if(fried)
+		return
+	anchored = TRUE // good luck getting this bad boy off
```
Comment (NonQueueingMatt) [inline code/game/objects/items/skrell.dm]: I don't know about this. If their head gets cut off then the tag will still be anchored once on the ground, meaning you can't move it.

Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
```

File: code/game/objects/items/skrell.dm [hunk 9]
```diff
@@ -153,3 +153,57 @@
...
+	canremove = FALSE
+	icon_state = "[initial(icon_state)]_active"
+	to_chat(wearer, span("warning", "\The [src] clamps down around your ear, releasing a burst of static before going silent. Something probes at your ear canal..."))
+	addtimer(CALLBACK(src, .proc/do_loyalty, wearer), 15)
+
+/obj/item/jargontag/proc/unclamp()
+	if(fried)
+		return
+	if(anchored && !canremove)
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
	if(!canremove)
```

File: code/game/objects/items/skrell.dm [hunk 10]
```diff
@@ -153,3 +153,57 @@
...
+		icon_state = initial(icon_state)
+		visible_message(span("warning", "\The [src] fizzles loudly, then clicks open!"))
+		anchored = FALSE
```
Comment (Geevies) [inline code/game/objects/items/skrell.dm]:
```suggestion
```

**Comment by Geevies** [comment]
!review

**Pull request merged**